pkg/migrate/migrate.go

```diff
@@ -48,42 +47,7 @@ type Options struct {
 	SetDefaults          bool
 	VerboseCopy          bool
 	SkipSourceValidation bool
-}
-
-// Cli uses CLI options to run Migrate
-func Cli() {
-	var options Options
-
-	flag.StringVar(&options.SourceSCName, "source-sc", "", "storage provider name to migrate from")
-	flag.StringVar(&options.DestSCName, "dest-sc", "", "storage provider name to migrate to")
-	flag.StringVar(&options.RsyncImage, "rsync-image", "eeacms/rsync:2.3", "the image to use to copy PVCs - must have 'rsync' on the path")
-	flag.StringVar(&options.Namespace, "namespace", "", "only migrate PVCs within this namespace")
-	flag.BoolVar(&options.SetDefaults, "set-defaults", false, "change default storage class from source to dest")
-	flag.BoolVar(&options.VerboseCopy, "verbose-copy", false, "show output from the rsync command used to copy data between PVCs")
-	flag.BoolVar(&options.SkipSourceValidation, "skip-source-validation", false, "migrate from PVCs using a particular StorageClass name, even if that StorageClass does not exist")
-
-	flag.Parse()
-
-	// setup k8s
-	cfg, err := config.GetConfig()
-	if err != nil {
-		fmt.Printf("failed to get config: %s\n", err.Error())
-		os.Exit(1)
-	}
-
-	clientset, err := k8sclient.NewForConfig(cfg)
-	if err != nil {
-		fmt.Printf("failed to create kubernetes clientset: %s\n", err.Error())
-		os.Exit(1)
-	}
-
-	output := log.New(os.Stdout, "", 0) // this has no time prefix etc
-
-	err = Migrate(context.TODO(), output, clientset, options)
-	if err != nil {
-		fmt.Printf("%s\n", err.Error())
-		os.Exit(1)
-	}
+	PodReadyTimeout      int
```

Comment:
This needs to be a time.Duration, not an int

Reply:
Fixed.